images/ipfailover/keepalived/lib/config-generators.sh

```diff
@@ -247,48 +227,47 @@ vrrp_instance ${instance_name} {
 #      generate_failover_configuration
 #
 function generate_failover_config() {
-  local vips=$(expand_ip_ranges "$HA_VIPS")
-  local interface=$(get_network_device "$NETWORK_INTERFACE")
-  local ipaddr=$(get_device_ip_address "$interface")
-  local port=$(echo "$HA_MONITOR_PORT" | sed 's/[^0-9]//g')
+  local vips=$(expand_ip_ranges "${HA_VIPS}")
+  local interface=$(get_network_device "${NETWORK_INTERFACE}")
+  local ipaddr=$(get_device_ip_address "${interface}")
+  local port=$(echo "${HA_MONITOR_PORT}" | sed 's/[^0-9]//g')
 
   echo "! Configuration File for keepalived
 
-$(generate_global_config "$HA_CONFIG_NAME")
-$(generate_script_config "$ipaddr" "$port")
-$(generate_vrrp_sync_groups "$HA_CONFIG_NAME" "$HA_VIPS")
+$(generate_global_config "${HA_CONFIG_NAME}")
```

Comment:
Probably out of the scope of this PR. Just wanted to confirm that this is the intended behavior.